scripts/bin/update-packages

```diff
@@ -212,12 +209,24 @@ _fetch_and_cache_tags() {
 			source "${i}"
 		done
 		set -eu
+
+		# PKG means regular version to be cached normally.
+		# SKIP means we know package is up to date and we can safely skip checks.
+		# GIT_C is for obtaining current package version, to be compared with version obtained with GraphQL
+
+		termux_github_api_get_tag() {
+			echo "GIT_C|${TERMUX_PKG_NAME}|${TERMUX_PKG_VERSION#*:}"
+			exit 0
+		}
```

Comment:
If this is a stub because we're using GraphQL for GitHub packages that should be explicitly stated with a comment.

Reply:
It is not a stub. It gets CURRENT package version.
It is needed in order to drop already up to date packages without doing `$(set +eu; source $pkg_dir/build.sh &> /dev/null; echo $TERMUX_PKG_VERSION)` one more time and to not wait for fetch&cache tags step to finish.

Reply:
I think this should have a comment explaining it either way because that is not obvious to me from a first read.

Reply:
Actually it has comment two lines above. I am not sure how to explain this better.

Reply:
Hmm maybe the issue isn't the comment but the name.
`LOCAL` instead of `GIT_C` might be more obvious.

Reply:
But it does not refer to GIT originating. And I do not want to increase label length because it will increase variable comparing overhead.

Reply:
I will do that though, it makes more sense.

Reply:
`LOCAL` and `GIT_C` are both 5 characters (that's partially why I suggested `LOCAL`)

Reply:
I mean I did not want to make it `GIT_CURRENT` or `GIT_LOCAL` or anything similar.

Reply:
Now should be fine.

Reply:
Oh yeah, I purely meant the "action" tag for the ad-hoc format we're using here.